online-shop/src/components/products/ProductList.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { IProduct } from '../../types';
+import { ControlButton } from '../utils/ControlButton';
+import { IconControlButton } from '../utils/IconControlButton';
+import { Product } from "./Product";
+import { Link } from 'react-router-dom';
+import { ORDERS_PATH } from '../../constants';
+
+export function ProductList({ data }: { data: IProduct[] }) {
+  return (
+    <div className='section'>
+      <nav className="navbar is-transparent">
+        <div className='navbar-start'>
+          <p className='subtitle is-5'>
+            <strong>Products</strong>
+          </p>
+        </div>
+        <div className="navbar-end">
+          <div className="field is-grouped">
+            <Link to={`${ORDERS_PATH}`}>
```

Comment:
Why use a string template (with the back ticks), when you can just do `<Link to={ORDERS_PATH}>`?